pkg/runtime/cli.go

```diff
@@ -72,6 +72,7 @@ func FromFlags() (*DaprRuntime, error) {
 	daprHTTPStreamRequestBody := flag.Bool("dapr-http-stream-request-body", false, "Enables request body streaming on http server")
 	daprGracefulShutdownSeconds := flag.Int("dapr-graceful-shutdown-seconds", -1, "Graceful shutdown time in seconds.")
 	enableAPILogging := flag.Bool("enable-api-logging", false, "Enable API logging for API calls")
+	enableBuiltinK8sSecretStore := flag.Bool("enable-builtin-k8s-secret-store", true, "Enable Builtin Kubernetes Secret Store")
```

Comment:
Default being `true` is great. It makes this change backwards compatible.

Reply:
If the default is `true`, shouldn't the flag be called `no-builtin-k8s-secret-store` (or `disable-builtin-k8s-secret-store`) with a default of `false`? Most UNIX CLIs do not use `true` for missing flags.
(Same for the annotation)

Reply:
@ItalyPaleAle @artursouza If disable-builtin-k8s-secret-store, with a default value of `false` makes better sense here to users, then I can change the flag/annotation accordingly. Is that the final decision here? Just want to confirm before I make the actual change.

Reply:
I would recommend the `--no-` option to turn the feature off. If the default is to have the feature on, that seems the most common pattern among CLIs.

Reply:
OK, let's change to `disable-builtin-k8s-secret-store`

Reply:
Please review the changes now.